pkgs/development/interpreters/python/mk-python-derivation.nix

```diff
@@ -77,6 +83,13 @@ python.stdenv.mkDerivation (builtins.removeAttrs attrs [
 
   postFixup = lib.optionalString (!dontWrapPythonPrograms) ''
     wrapPythonPrograms
+  '' + lib.optionalString removeBinBytecode ''
+    if [ -d "$out/bin" ]; then
+      pushd "$out/bin"
+      rm -rf __pycache__            # Python 3
+      find . -name ".pyc" -delete   # Python 2
+      popd
+    fi
```

Comment:
pushd/popd result in some surprising noise in the console. `-name .pyc` does not match anything. Could you use `rm -rf "$out/bin/__pycache__"` and `find "$out/bin" -type f -name "*.pyc" -delete` instead?